### PR TITLE
v1.1

### PR DESCRIPTION
**Pull request by thewchan**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by thewchan** [comment]
@conda-forge-admin, please rerender

**Comment by thewchan** [comment]
@conda-forge-admin, please rerender

**Comment by thewchan** [comment]
@dhirschfeld @dgasmith @marcelotrevisani @vallsv requesting review and merge. Thanks!

**Comment by vallsv** [comment]
Hi @thewchan

I haven't check deeply, but i suggest to remove the dependency to `mkdocs-material`.

- `mkdocs-material` is the one which request `mkdocs-material-extensions`
- And you can see that there is already a dependency to `mkdocs-material` with the  `run_constrained`

What do you think?

**Comment by vallsv** [comment]
If you want access now to the repo, feel free to create a new PR with your user name only, ill merge it directly.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,39 +1,47 @@
 {% set name = "mkdocs-material-extensions" %}
-{% set version = "1.0.3" %}
+{% set version = "1.1" %}
```
Comment (dhirschfeld) [inline recipe/meta.yaml:2]:
```suggestion
{% set version = "1.1.1" %}
```
https://github.com/facelessuser/mkdocs-material-extensions/releases/tag/1.1.1

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,39 +1,47 @@
...
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: bfd24dfdef7b41c312ede42648f9eb83476ea168ec163b613f9abd12bbfddba2
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/mkdocs_material_extensions-{{ version }}.tar.gz
+  sha256: 96ca979dae66d65c2099eefe189b49d5ac62f76afb59c38e069ffc7cf3c131ec
```
Comment (dhirschfeld) [inline recipe/meta.yaml:11]:
```suggestion
  sha256: 9c003da71e2cc2493d910237448c672e00cefc800d3d6ae93d2fc69979e3bd93
```
https://pypi.org/project/mkdocs-material-extensions/1.1.1/#copy-hash-modal-4ff14440-53c7-4e5c-a24f-f2b42fb00435

**Comment by thewchan** [comment]
> Hi @thewchan
> 
> I haven't check deeply, but i suggest to remove the dependency to `mkdocs-material`.
> 
>     * `mkdocs-material` is the one which request `mkdocs-material-extensions`
> 
>     * And you can see that there is already a dependency to `mkdocs-material` with the  `run_constrained`
> 
> 
> What do you think?

@vallsv The dependencies for `mkdocs-material` is only for testing, not for running. The version cited there is specified upstream.

File: recipe/meta.yaml (hunk 1 above)
Comment (thewchan) [reply recipe/meta.yaml:2]: We can merge in this one (1.1) first, then the bot will pick up 1.1.1 next?

Comment (dhirschfeld) [reply recipe/meta.yaml:2]: I don't see too much point in publishing a package which has already been superseded but if you prefer to do it that way, that's fine. As a maintainer you'll be able to merge version 1.1.1 yourself.

**Review by dhirschfeld — APPROVED** [review]

**Comment by vallsv** [comment]
Sounds like every body is happy. I have merged the changes.

**Pull request merged**